concent_api/core/validation.py

```diff
@@ -106,6 +106,23 @@ def validate_list_of_identical_task_to_compute(list_of_task_to_compute: List[mes
     return True
 
 
+def validate_list_of_identical_report_computed_task(list_of_report_computed_task: List[message.tasks.ReportComputedTask]):
```

Comment:
How about making `validate_list_of_identical_task_to_compute` generic instead of introducing new one ? You have to only:
- pass `message_type` to use it in lines 85, 93 and 96
- change names to more generic (`golem_message` instead `task_to_compute`)

Reply:
Done